needs-formatting.md

```diff
@@ -1,47 +1,54 @@
+Based on fossil evidence from Russia and Bulgaria, the first known members of the family Procyonidae lived in Europe in the late Oligocene about 25 million years ago. Similar tooth and skull structures suggest procyonids and weasels share a common ancestor, but molecular analysis indicates a closer relationship between raccoons and bears.  
+  
+ After the then-existing species crossed the Bering Strait at least six million years later in the early Miocene, the center of its distribution was probably in Central America. Coatis (Nasua and Nasuella) and raccoons (Procyon) have been considered to share common descent from a species in the genus Paranasua present between 5.2 and 6.
+ 
+ 0 million years ago. This assumption, based on morphological comparisons of fossils, conflicts with a 2006 genetic analysis which indicates raccoons are more closely related to ringtails. Unlike other procyonids, such as the crab-eating raccoon (Procyon cancrivorus), the ancestors of the common raccoon left tropical and subtropical areas and migrated farther north about 2.5 million years ago, in a migration that has been confirmed by the discovery of fossils in the Great Plains dating back to the middle of the Pliocene. Its most recent ancestor was likely Procyon rexroadensis, a large Blancan raccoon from the Rexroad Formation characterized by its narrow back teeth and large lower jaw.
 
 <!-- Change the link below to be an image. Include descriptive alternate text. -->
 
-https://curiodyssey.org/wp-content/uploads/bb-plugin/cache/Mammals-Raccoon-square.jpg
+![](https://curiodyssey.org/wp-content/uploads/bb-plugin/cache/Mammals-Raccoon-square.jpg)
```

Comment:
remember to add 'alt text' in the `[]` for screen readers and in case the image does not load